build.gradle

```diff
@@ -18,7 +18,7 @@ sourceCompatibility = JavaVersion.VERSION_1_7
 targetCompatibility = JavaVersion.VERSION_1_7
 
 group = 'com.google.cloud.tools'
-version = '0.1.2-SNAPSHOT'
+version = '0.2.0-SNAPSHOT'
```

Comment:
mostly just because I change a lot of the internals and would like to play around with it for a week before switching to 1.0.0